### PR TITLE
spoopy skeletons 💀

### PR DESCRIPTION
**Pull request by davidleger95**
## Change

Screen Shot 2020-04-30 at 12 41 44 PM

### CODE REVIEW

File: src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx [hunk 1]
```diff
@@ -4,24 +4,29 @@ import MeteredFeature from 'components/shared/MeteredFeature';
 import ConfigurableFeature from 'components/shared/ConfigurableFeature';
 import PlanCard from 'components/shared/PlanCard';
 import CostDisplay from 'components/shared/CostDisplay';
+import SkeletonPlanSelector from 'components/shared/SkeletonPlanSelector';
+import { FeatureMap } from 'utils/plan';
```
Comment (davidleger95) [inline src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx:8]: Did a bit of plan refactoring while i was at it

File: src/styles/elements/_skeleton.scss [hunk 2]
```diff
@@ -70,3 +72,10 @@ manifold-subscription-create [data-is-loading] {
 manifold-subscription-list [data-is-loading] {
   @include Skeleton();
 }
+
+.ManifoldSubscription__SkeletonText {
+  @include mercury.Skeleton;
+  * {
+    filter: opacity(0);
+  }
```
Comment (davidleger95) [inline src/styles/elements/_skeleton.scss:80]: This is a good way to make sure contents never leak through on a skeleton

File: src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx (hunk 1 above)
Comment (dangodev) [reply src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx:8]: Do these absolute imports work when built? I was thinking they broke when you consumed this in TypeScript and that everything needed to be relative

File: src/styles/elements/_skeleton.scss [hunk 3]
```diff
@@ -70,3 +72,10 @@ manifold-subscription-create [data-is-loading] {
...
+}
```
Comment (dangodev) [inline src/styles/elements/_skeleton.scss:81]: 💀

File: src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx (hunk 1 above)
Comment (davidleger95) [reply src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx:8]: the CI all passed so many it's been fixed 🤷‍♂️ I'll check it out

Comment (dangodev) [reply src/components/manifold-subscription-details/components/Edit/PlanSelector.tsx:8]: Yeah IIRC it works in JS, but in a TS app (React + TS) it blows up. But worth testing. May only be for certain things like the generated GraphQL types

**Pull request merged**